.github/workflows/ci.yml

```diff
@@ -28,7 +28,7 @@ jobs:
       image: registry.gitlab.com/finestructure/spi-base:0.14.1
     services:
       postgres:
-        image: postgres:13.8-alpine
+        image: postgres:13.11-alpine
```

Comment:
I was confused at first, but if I'm with you this is just a minor version bump in 13 rather than upgrading to 15 as mentioned in #2753?